__test__/cli-test/content-types/with-regex.ts

```diff
@@ -0,0 +1,12 @@
+import { contentType } from '@episerver/cms-sdk';
+
+export const contentTypeWithRegex = contentType({
+  key: 'CT_with_regex',
```

Comment:
Same here, but here the key, display name, and filename also have "regex".

Reply:
Good catch. This was even worse because I forgot to add an actual regex in the content type property 🤦